tools/bin/sstableloader

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "x$CASSANDRA_INCLUDE" = "x" ]; then
```

Comment:
@tolbertam please check `redhat/cassandra.spec`. I know that `bin/sstableloader` points to this one, I just wonder what changes, if anything, if we introduce this script among tools.

Reply:
tools are in a different package a user has to install so when we move sstableloader among tools, then, suddenly, the script in `bin/sstableloader` will point to a non-existing `tools/bin/sstableloader`.

Reply:
ah, this is not something I had thought about yet, I'll see what I can do about this.

Reply:
resolving conversation as in other comments noted that was able to make both the debian and redhat builds work :+1: